src/get_covid_hosp_data.R

```diff
@@ -156,7 +156,9 @@ if (!target_data) {
   output_folder_path <- fs::path(
     base_hub_path, "weekly-summaries", reference_date
   )
-  output_filename <- paste0(reference_date, "_truth-data.csv")
+  output_filename <- paste0(
+    reference_date, "_covid_target_hospital_admissions_data.csv"
+  )
```

Comment:
This is fine for now, but for future reference good to specific string-formatting of dates explicitly or at least use `glue`
```suggestion
  output_filename <- paste0(
    reference_date, "_covid_target_hospital_admissions_data.csv"
  )
```